dds/idl/marshal_generator.cpp

```diff
@@ -2162,9 +2162,6 @@ namespace {
     if (repr.xml) {
       values.push_back("DDS::XML_DATA_REPRESENTATION");
     }
-    if (repr.unaligned_cdr) {
-      values.push_back("UNALIGNED_CDR_DATA_REPRESENTATION");
```

Comment:
If we're not using it anymore, I think `OpenDDS::UNALIGNED_CDR_DATA_REPRESENTATION` can be removed from `DdsDcpsCore.idl`.

Reply:
It's removed now.